python-avd/pyavd/api/fabric_documentation/__init__.py

```diff
@@ -1,6 +1,28 @@
 # Copyright (c) 2024-2025 Arista Networks, Inc.
 # Use of this source code is governed by the Apache License 2.0
 # that can be found in the LICENSE file.
+
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class DigitalTwinFabricDocumentationActLink:
+    connection: tuple[str | None, str | None]
+
+
+@dataclass(frozen=True)
+class DigitalTwinFabricDocumentationActNodeType:
```

Comment:
I don't think we need to include the whole context in the class names. The class itself is not reliant on the context.
Also this doesn't seem like a type, but more like details / settings for one node type.
```suggestion
class ActNodeTypeSettings:
```

Reply:
Applied